packages/core/src/common/keybinding.ts

```diff
@@ -26,6 +26,13 @@ export interface Keybinding {
     readonly accelerator?: Accelerator;
 }
 
+export interface RawKeybinding {
```

Comment:
Could we just get rid of KeyCode in the `Keybinding` and use that one instead?

Reply:
If by "that one" you meant the string representation, I think it's doable, but there's going to be refactoring the monaco part. From what I understand having that KeyCode structure makes it easier to rebind the monaco commands to theia commands. Worth investigating for sure, as I think one structure for everything everywhere is better.

Reply:
Ok let's do that later, then.